### DISCUSSION TITLE
My first Photoobooth, and my next

### DISCUSSION THREAD

**Discussion by find-usdk** [photobooth-app/photobooth-app#380] [Show and tell]
This is my first attemt to make a Photobooth.

Running on a Rasperry Pi 5  and an old touch screen.
Works great, and have add a LED light on top of the box.

![IMG_3815](https://github.com/user-attachments/assets/22424ee9-8c14-4246-becb-b22232645151)
![IMG_3899](https://github.com/user-attachments/assets/33814729-796c-4428-8ef2-1843463d5a8b)

Video of the LED light: https://github.com/user-attachments/assets/43a3ebc5-15bd-4ac9-a1c2-789e67c284c0

I'm currently testing it on this little cheap 10" touch screen with build in Rasperry Pi 5 (arround 250€) : 
[HMI3010-7datasheet-en.pdf](https://github.com/user-attachments/files/18178430/HMI3010-7datasheet-en.pdf)

**Comment by PhiHar** [comment]
Nice, how did. you manage to Control the led light?

**Reply by find-usdk** [reply]
I have used a button with 2 contact-set, so when you press the button i use the 1 contact-set to activate the GPIO (Take picture), and the other contact-set to activate a electric timer for the light.

![image](https://github.com/user-attachments/assets/9f87b27f-73c7-429d-80a5-5160a619ad41)

The timer is then set to the same time as the countdown of the picture i taken.

**Comment by rektosaure** [comment]
Hi, where did you buy this tripod? Thanks

**Reply by find-usdk** [reply]
Hi
Sorry for the late reply.
It´s one i have got for free from a mate.
But it´s just a speaker tripod like this: 
https://www.thomann.de/intl/fun_generation_speaker_stand_pair.htm?shp=eyJjb3VudHJ5IjoiZGsiLCJjdXJyZW5jeSI6NiwibGFuZ3VhZ2UiOjJ9&reload=1&glp=1&gad_source=1&gclid=CjwKCAiAwaG9BhAREiwAdhv6Ywtudq0ZjrIADznW5Rdc0IVhlBs77j_Jr0vU0B20BWJmf4OrNgQOOxoCEPwQAvD_BwE

And then this mount in the Selfiebox: 
https://www.thomann.de/dk/adam_hall_sm_702.htm?glp=1&gad_source=1&gclid=CjwKCAiAwaG9BhAREiwAdhv6YzaJxcQrU54VXCD3ouFLu0WoOu5El4Jbun7SSvxw6Y9OcVy5ZLSkzBoCfhIQAvD_BwE

**Reply by rektosaure** [reply]
thank you 👍

**Reply by find-usdk** [reply]
Another time, I would probably have chosen a super clamp instead of the other mount, as it wouldn’t be able to rotate when operating it.

https://www.thomann.de/dk/manfrotto_035264_super_clamp_pack.htm
